ci/test.sh

```diff
@@ -102,5 +102,13 @@ convox ps stop $ps -a httpd
 convox ps -a httpd | grep -v $ps
 convox deploy -a httpd
 
+# timers
+sleep 30
```

Comment:
It seems that 30 secs is not enough https://github.com/convox/convox/runs/8207383225?check_suite_focus=true
maybe using 2m?

Reply:
Actually @Twsouza, this is another issue that I'll fix in the next commit. 30 seconds is enough. Well, at least, so far.